JS/Tip_calculator/tip-calculator.js

```diff
@@ -0,0 +1,5 @@
+const costOfMeal = 30;
+const tip = 0.1;
+let tipAmount = costOfMeal * tip;
+let totalCost = tipAmount + costOfMeal;
+console.log("The total cost of your bill is €" + totalCost);
```

Comment:
The message should look like:
`The total cost of your bill is: 33 euros` or `The total cost of your bill is: 33 €`